tst/EnergyPlus/api/test_OutputFiles.py

```diff
@@ -86,27 +93,26 @@ def test_proper_run(self):
         Tests the case where the simulation ends properly and EndEnergyPlus is
         called
         """
-        with tempfile.TemporaryDirectory() as tempdir:
-            print(tempdir)
-            cmd_args = shlex.split(f'-d {tempdir} -D @IDF_FILE@')
-            return_code = self.api.runtime.run_energyplus(self.state, cmd_args)
-            self.assertEqual(return_code, 0)
-            out_files = gb.glob(os.path.join(tempdir, "*"))
-            # We need to find out files, and we expect NONE of them to be empty
-            self.assertGreater(len(out_files), 10)
-            empty_files = list(filter(lambda f: os.stat(f).st_size == 0,
-                                      out_files))
-            msg = f"Did not expect empty files, but found {len(empty_files)}:"
-            for f in empty_files:
-                msg += f"\n * {os.path.relpath(f, tempdir)}"
-            self.assertFalse(empty_files, msg)
-
-            # Check that the eplusout.err is not truncated, that the last line
-            # is what we expect it to be
-            with open(os.path.join(tempdir, 'eplusout.err'), 'r') as f:
-                lines = f.read().splitlines()
-            self.assertTrue(lines, "eplusout.err is empty!")
-            self.assertIn("EnergyPlus Completed Successfully", lines[-1])
+        tempdir = tempfile.mkdtemp()
```

Comment:
I was getting some strange behavior from the TemporaryDirectory being used as a Context Manager.  In the `with` clause, the directory is cleaned up at the end, but I was getting issues with EnergyPlus not being able to find files and unable to clean up.  I'm not sure who was at fault, but I just made it a normal variable.  It will leave 2 temp directories with a handful of small output files each time you run this API test locally.  Should not be a big deal.  And we can look into it later if necessary.